src/main/java/org/opensearch/ad/transport/DeleteModelTransportAction.java

```diff
@@ -113,7 +114,15 @@ protected DeleteModelNodeResponse nodeOperation(DeleteModelNodeRequest request)
         String adID = request.getAdID();
         LOG.info("Delete model for {}", adID);
         // delete in-memory models and model checkpoint
-        modelManager.clear(adID);
+        modelManager
+            .clear(
+                adID,
+                ActionListener
+                    .wrap(
+                        r -> LOG.info("Deleted model for [{}] with response [{}] ", adID, r),
+                        e -> LOG.error("fail to delete model for " + adID, e)
```

Comment:
super minor: capitalize to `Fail` for consistency

Reply:
yeah, changed